script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -530,6 +530,7 @@ It seems I've made her realize what she's put me through up to this point.
 [sha:705937fed4df648da49e5c2acbd8d0d60ebff9a4]{
 -- Page 78, Offset 8775.
 -- 「―――うそ。あんなに手馴れてて、あれが初めてだったっていうの、志貴……!?」
+// Light wants this line fixed
```

Comment:
Sorry light but this line is good

Reply:
Clauses are a little disconnected, I suppose?
>No way. You're telling me that was your first time, Shiki? Even though you were that skilled at it?!

I could see a change like this, though I agree it's a pretty minor issue. Wanna do my boy (?) proud.

Reply:
Hmm, I still kinda prefer current. A bit more natural speech-y, especially that last segment. Also have a mild preference for keeping names in roughly the same places where possible to match VA

Reply:
I'll give Light 1 day to chime in, lol.

Reply:
I'd prefer to just remove the name 'Shiki' and keep the current version then rearrange it like that.

Reply:
I don't understand what you mean by that